### DISCUSSION TITLE
Hachette cubes in Italy

### DISCUSSION THREAD

**Discussion by n-harari** [oyooyo/audiocube#20] [General]
Hello, 
just wanted to let you know that these audiobooks were sold by Hachette in Italy as well:
https://figlimoderni.it/2020/01/audiofiabe-la-nuova-collana-di-hachette.html
The series was suspended though.
I have all the books and sd cards, however the audio box broke down (it won't charge anymore) and I am desperate to find a substitute. Any idea of what device I can use? It seems impossible to find the Hachette cube on Ebay or other websites, I tried looking for the Mexican, Spanish, German version at no avail. 
Thanks for helping out a total newbie!

**Comment by domiwar** [comment]
Hi
I can suggest you to use a power bank attached externally or change the battery inside.
@oyooyo The Italy  version is the blu one, before find this repo, I managed by myself thought mifare classic app and an android phone to generate all the tag that I had missed and worked good. Now tanks to the software in this repo I had also the mp3 version to listen in every device.

**Comment by adrianbraitoru** [comment]
> Hello, just wanted to let you know that these audiobooks were sold by Hachette in Italy as well: https://figlimoderni.it/2020/01/audiofiabe-la-nuova-collana-di-hachette.html The series was suspended though. I have all the books and sd cards, however the audio box broke down (it won't charge anymore) and I am desperate to find a substitute. Any idea of what device I can use? It seems impossible to find the Hachette cube on Ebay or other websites, I tried looking for the Mexican, Spanish, German version at no avail. Thanks for helping out a total newbie!

Hello,

Can you please help me with the files from the micro SD card?

Thanks,
Adrian

**Reply by domiwar** [reply]
Whit the software of this repo you can decrypt the mp3 and put inside a portable mp3 player, and with a cable or bluetooth play in any musyc speaker

**Reply by Xaroulini** [reply]
with Smp2Mp3

**Comment by Xaroulini** [comment]
Hello, can you upload the files from SD card, please? I search it...

**Comment by Xaroulini** [comment]
> Hello, just wanted to let you know that these audiobooks were sold by Hachette in Italy as well: https://figlimoderni.it/2020/01/audiofiabe-la-nuova-collana-di-hachette.html The series was suspended though. I have all the books and sd cards, however the audio box broke down (it won't charge anymore) and I am desperate to find a substitute. Any idea of what device I can use? It seems impossible to find the Hachette cube on Ebay or other websites, I tried looking for the Mexican, Spanish, German version at no avail. Thanks for helping out a total newbie!

Hello, could you give the files of sd card at Italy? I search it, please!!??? Can you upload as it is inside the memory card? please!!!!

**Comment by domiwar** [comment]
Have you tryed with attached powerbank? anyway he battery inside is pretty
simple to fix.
